docs/providers/coinbasewallet.md

```diff
@@ -0,0 +1,33 @@
+# Coinbase Wallet
+_(formerly WalletLink)_
+
+1. Install Provider Package
+
+```bash
+npm install --save @coinbase/wallet-sdk
+
+# OR
+
+yarn add @coinbase/wallet-sdk
+```
+
+2. Set Provider Options
+
+```typescript
+import CoinbaseWalletSDK from '@coinbase/wallet-sdk';
+
+const providerOptions = {
+  coinbasewallet: {
```

Comment:
maybe `coinbaseWallet`?

Reply:
per `ADDING_PROVIDERS.md`

> the id should be lowercase

b/c it'll map to the filename, also in lowercase.